runtime-watcher/go.mod

```diff
@@ -2,14 +2,11 @@ module github.com/kyma-project/runtime-watcher/skr
 
 go 1.23.2
 
-// TODO: https://github.com/kyma-project/runtime-watcher/issues/402
-replace golang.org/x/text v0.17.0 => golang.org/x/text v0.19.0
-
 require (
 	github.com/go-logr/logr v1.4.2
 	github.com/go-logr/zapr v1.3.0
 	github.com/google/uuid v1.6.0
-	github.com/kyma-project/lifecycle-manager/api v0.0.0-20240918085249-66f3802ce0ae
+	github.com/kyma-project/lifecycle-manager/api v0.0.0-20241014091215-4f6df7c7b2ae
```

Comment:
to reviewer: again, replace for `x/text` didn't work as expected. Saw that it may be coming from `lifecycle-manager/api` with outdated depenency to OCM 0.13.0. Tried updating this which seems to work.

Reply:
Not sure if I get you exactly.

The KLM updated fixed the `x/text` version in `runtime-watcher`. For `listener`, we don't have a KLM dependency. Therefore I did the replacement. Are you saying that we should remove the replacement completely, or replace to a version of `x/net` that is using `x/text` `0.18.0`, and not `0.19.0`?